index.html

```diff
@@ -46,778 +46,778 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
+                        <td>
+                            <ul>
+                                <li> <a href="https://arxiv.org/abs/1706.03741"> Deep reinforcement learning from
+                                        human preferences.</a> Christiano et al. (2017)</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 5</b><br />
+                            Wed, April 30
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">RL for LLMs: Preference
+                                Optimization</a>
+                        </td>
+                        <td>
+                        </td>
+                        <td>
+
+                            <ul>
+                                <li> <a href="https://arxiv.org/abs/2305.18290"> Direct Preference Optimization: Your
+                                        Language Model is Secretly a Reward Model.
+                                    </a> Rafailov et al. (2023)</li>
+                            </ul>
+
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 5</b><br />
+                            Fri, May 2
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">RL for LLMs: Reasoning</a>
+                        </td>
+                        <td>
+                            <span class="label label-due text-base">Due</span></i><b> HW2 Due</b><br>
+                            HW3 Out (Offline RL)
+                        </td>
+                        <td>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><b>Week 6</b><br />
+                            Wed, May 7
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">Model-based RL</a>
+                        </td>
+                        <td>
+                        </td>
+                        <td>
+                            <ul>
+                                <li> <a href="https://arxiv.org/abs/1906.08253">
+                                        When to Trust Your Model: Model-Based Policy Optimization
+                                    </a>
+                                    Janner et al. (2019)</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><b>Week 6</b><br />
+                            Fri, May 9
+                        </td>
+                        <td><i><span class="label label-guest text-base">Lecture</span></i>
```

Comment:
the label should be label-lecture not label-guest. Could you make sure all of the label colors are consistent?